Content.Server/DeltaV/StationEvents/Events/GlimmerMiteSpawnRule.cs

```diff
@@ -0,0 +1,58 @@
+using System.Linq;
+using Robust.Shared.Random;
+using Content.Server.GameTicking.Rules.Components;
+using Content.Server.NPC.Components;
+using Content.Server.Psionics.Glimmer;
+using Content.Server.StationEvents.Components;
+using Content.Shared.Psionics.Glimmer;
+using Content.Shared.Abilities.Psionics;
+
+namespace Content.Server.StationEvents.Events;
+
+internal sealed class GlimmerMiteRule : StationEventSystem<GlimmerMiteRuleComponent>
+{
+    [Dependency] private readonly IRobustRandom _robustRandom = default!;
+    [Dependency] private readonly GlimmerSystem _glimmerSystem = default!;
+
+    private static readonly string MitePrototype = "MobGlimmerMite";
+
+    protected override void Started(EntityUid uid, GlimmerMiteRuleComponent component, GameRuleComponent gameRule, GameRuleStartedEvent args)
+    {
+        base.Started(uid, component, gameRule, args);
+
+        var glimmerSources = EntityManager.EntityQuery<GlimmerSourceComponent, TransformComponent>().ToList();
+        var normalSpawnLocations = EntityManager.EntityQuery<VentCritterSpawnLocationComponent, TransformComponent>().ToList();
+        var hiddenSpawnLocations = EntityManager.EntityQuery<MidRoundAntagSpawnLocationComponent, TransformComponent>().ToList();
+
+        var baseCount = Math.Max(1, EntityManager.EntityQuery<PsionicComponent, NpcFactionMemberComponent>().Count() / 10);
```

Comment:
dont need EntityManager. due to proxy

Reply:
Do I just delete these lines then?

Reply:
no remove the `EntityManager.` before each EntityQuery call

Reply:
Ah okay, thanks. I'm just going to go through each of these comments one by one, btw. Thanks for the help!